mofiebiger/ember-trivia-game
# Ember trivia general layout

# ✨ What Changed & Why


## 👹 Before

## 😇 After

<img width="1680" alt="Screenshot 2020-04-07 at 15 13 08" src="https://user-images.githubusercontent.com/43208685/78679608-5c051380-78e2-11ea-91fd-6042bf9362c5.png">

## 🌈 Testing

1. Visit [Netlify-Site](https://loving-nightingale-e3329b.netlify.com) or [S3-Site](http://mo-ember-trivia-bucket.s3-website-eu-west-1.amazonaws.com)
1. Observe new header and footer components rendered on page
